cls/SourceControl/Git/Settings.cls

```diff
@@ -394,4 +395,25 @@ Method ConfigureBinPath(ByRef path As %String) As %Boolean
     return 1
 }
 
+Method RetrieveDefaults() As %Boolean
+{
+    do ##class(%zpkg.isc.sc.git.Defaults).GetDefaultSettings(.settings)
+    set iterator = settings.%GetIterator()
+    while iterator.%GetNext(.key, .value) {
+        set $property($this,key) = value
+    }
+    return $$$OK
+}
+
+Method SaveDefaults() As %Boolean
+{
+    set defaults = {}
+    set items = $lb("gitBinPath", "namespaceTemp", "pullEventClass", "percentClassReplace", "environmentName", "systemBasicMode", "defaultMergeBranch", "mappedItemsReadOnly", "compileOnImport")
```

Comment:
namespaceTemp shouldn't be a system-level default! This will always differ per namespace as things stand. Even once we add support for instance-wide, this is going to be more complex to configure (probably).